app/locales/en.json

```diff
@@ -171,5 +172,8 @@
   "I used to smoke": "I used to smoke",
   "I have never smoked": "I have never smoked",
   "What is your current body temperature? (Celcius)": "What is your current body temperature in Celcius?",
-  "This field is optional": "This field is optional"
+  "This field is optional": "This field is optional",
+  "Close": "Close",
+  "Contribute now": "Contribute now",
+  "Help us uncover and predict the corona spread in your area": "Help us <span>uncover</span> and <span>predict</span> the corona spread in your area"
```

Comment:
I think we should avoid having markup in our texts, it's better to split it up into multiple texts

Reply:
I agree. With this one it's really hard to do though. The span does not have any styling in the translations, I add that in the app, but I do not really see a better way. Could to tagline_1, tagline_2, ... but then you don't know which word to highlight. I think this translation is the exception to the rule.

Reply:
Other places we did something like this:
```
<%= __('Help us') %> <span><%= __('uncover') %></span> <%= __('and') %> <span><%= __('predict') %> <%= __('the corona spread in your area') %>
```

Reply:
That would work if the order of this highlighted words are the same in languages right? In Dutch the highlighter words are at the end.

Reply:
We could do `"Help us": "Help ons de verspreiding van het coronavirus te ", "uncover": "ontdekken", "and": "en", "predict": "voorspellen", "the corona spread in your area": "in uw omgeving"`. If you have a strong preference, let's do it. If not, let's keep it.

Reply:
I do think translators with other language will not understand these are connected and therefor fail in translation, though. It's also not a biggy for me.